tests/fixtures/vector_io.py

```
@@ -350,3 +365,116 @@ def get_pgvector_deployment_template() -> Dict[str, Any]:
             "volumes": [{"name": "pgdata", "emptyDir": {}}],
         },
     }
+
+
+@pytest.fixture(scope="class")
+def qdrant_deployment(
+    unprivileged_client: DynamicClient,
+    unprivileged_model_namespace: Namespace,
+    qdrant_secret: Secret,
```

Ruff (0.14.11)

Unused function argument: `qdrant_secret`

(ARG001)